package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rebilly-js-sdk",
-  "version": "17.2.0",
+  "version": "18.0.0",
```

Comment:
Seems like it should not be a major release. Semver2: major release is like a breaking change. 

https://semver.org/

Reply:
Fixed